### PR TITLE
feat: fixed an issue with video modal

### PR DESCRIPTION
**Pull request by shaheenfazim**
fixed issue where when video modal when closed youtube video still playing behind the background.
+ added counter animation to statistic values.

### CODE REVIEW

File: pages/crafts/crafts.html [hunk 1]
```diff
@@ -365,7 +372,7 @@ <h6 class="mb-3">
           <div class="modal fade" id="intro-modal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
             <div class="modal-dialog modal-xl  modal-dialog-centered" role="document">
               <div class="modal-content" style="background: none;border: none;">
-                <div class="modal-body" style="border-radius: 5px;margin-top: 50px;padding: 0;">
+                <div class="modal-body" style="border-radius: 5px;margin-top: 5vh;padding: 0;" id="yt-player">
 
                   <div class="embed-responsive embed-responsive-16by9">
                     <iframe class="embed-responsive-item" style="height: 70vh;width: 100%;" src="https://www.youtube.com/embed/zBf7DOtWRP8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (nandgator) [inline pages/crafts/crafts.html:378]: Bro, the video's is not loading (on local page):

image

Try with this link https://www.youtube.com/watch?v=Jwgf3wmiA04

**Review by nandgator — COMMENTED** [review]
Nice, numbers count up effect.

File: pages/crafts/crafts.html (hunk 1 above)
Comment (shaheenfazim) [reply pages/crafts/crafts.html:378]: yeah same its the problem on the youtube side

Comment (shaheenfazim) [reply pages/crafts/crafts.html:378]: they have some policy preventing us from running in local so.

Comment (nandgator) [reply pages/crafts/crafts.html:378]: oh.. okay.. so shall I merge this.. everything else look fine for now..

Comment (shaheenfazim) [reply pages/crafts/crafts.html:378]: yes

**Comment by shaheenfazim** [comment]
( ͠👁️ ͜ʖ ͡👁️)

**Comment by shaheenfazim** [comment]
Yup the issue still there

**Comment by nandgator** [comment]
Sync with upstream @NehaAJ @codedstrings @Sreesankar0 @sreehari666 :)

**Pull request merged**